example/build.gradle

```diff
@@ -29,6 +29,10 @@ android {
         exclude 'LICENSE.txt'
     }
 
+    lintOptions {
+        disable 'MissingTranslation'
```

Comment:
We don't have full translations on things for the example app. Would love to change that someday, but for now this was causing an error with the newly supercharged linter.